cob_bringup/components/canopen_base.launch

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <launch>
 	<arg name="robot"/>
-	<arg name="component_name" value="base"/>
+	<arg name="component_name" default="base"/>
```

Comment:
These should be values, I just kept the argument for having the config at the top and not spreaded.